Homework_11/CircusManagement/src/Circus.java

```diff
@@ -0,0 +1,89 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+
+public class Circus<T extends CircusPerformer> {
+    private ArrayList<T> performers = new ArrayList<>();
+
+
+    public void addPerformer(T performer) {
+        this.performers.add(performer);
+
+    }
+
+    public void removePerformer(T performer) {
+        this.performers.remove(performer);
+    }
+
+
+    public void listPerformers() {
+        System.out.println("This list of all our performers:");
+        performers.forEach(worker -> System.out.println(worker));
```

Comment:
`.forEach()`  is well used here, pretty laconic isn't it? 👍 

Love it when one line does the thing and it's simple to understand at the same time for someone, who reads the code